test/Interop/SwiftToCxx/functions/swift-operators.swift

```diff
@@ -0,0 +1,33 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend %s -typecheck -module-name Operators -clang-header-expose-decls=all-public -emit-clang-header-path %t/operators.h
+// RUN: %FileCheck %s < %t/operators.h
+
+// TODO: %check-interop-cxx-header-in-clang(%t/operators.h)
```

Comment:
The emitted header still isn't fully valid in C++:
```
test/Interop/SwiftToCxx/functions/Output/swift-operators.swift.tmp/operators.h:620:120: error: no member named '_impl_IntBox' in namespace 'Operators::_impl'
  return _impl::$s9Operators1soiys5Int32VAA6IntBoxV_AFtF(_impl::swift_interop_passDirect_Operators_uint32_t_0_4(_impl::_impl_IntBox::getOpaquePointer(lhs)), _impl::swift_interop_passDirect_Operators_uint32_t_0_4(_impl::_impl_IntBox::getOpaquePointer(rhs)));
                                                                                                                ~~~~~~~^

```
since the definition of `_impl_IntBox` is emitted after the function body.